File: SPA19/EGIS/src/app/toims/models/revenuecodeparent.ts
export interface IRevenueCodeParent {
    id: number;
    description: string;
    code : string;
    kind : number;
    seqNo : number;
 }

 export type createRevenueCodeParentDto =Omit<IRevenueCodeParent,'code'|'description'|'SeqNO'|'kind'>;

File: EGIS_Management/Domain/Entities/toims/RevenueCodeParent.cs
using System.ComponentModel.DataAnnotations;

namespace Domain.Entities
{
    public class RevenueCodeParent
    {
        [Key]
        public int Id { get; set; }
        public int SeqNo { get; set; }
        public string Kind { get; set; } = string.Empty;
        public string Code { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public DateTime CreatedAt { get; set; } = DateTime.Now;
    }
}
